apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V66__new_schema_snf_claims_fix.sql

```diff
@@ -0,0 +1,522 @@
+-- NEW_SCHEMA_SNF_CLAIMS_FIX.SQL
+-- flyway migration for SNF_CLAIMS_NEW and SNF_CLAIM_LINES_NEW tables into
+-- a new schema structure; previus flyway script, V63__nw_schema_snf_claims.sql
+-- whcich erroneously defined columns in the snf_claim_lines_new table
+-- as an integer. This script fixes that mistake and does a complete data
+-- migration of SNF claims data because the _new tables would have missed
+-- a weekend's ETL propcessing.
+--
+-- Once current table data is migrated to new table name/structtre,
+-- a subsequent PR will be deployed that changes the ORM model(s)
+-- for SNF table(s).
+
+-- truncate _new SNF claims tables in order to reload
+${logic.hsql-only}  TRUNCATE table snf_claim_lines_new;
+${logic.hsql-only}  TRUNCATE table snf_claims_new;
+${logic.psql-only}  TRUNCATE snf_claim_lines_new, snf_claims_new;
+
+-- fix the columns that were erroneously defined
+-- Values from Excel spreadsheet are:
+-- carrier_claims table:
+--     CLM_UTLZTN_DAY_CNT           NUM  *   *
+--     BENE_TOT_COINSRNC_DAYS_CNT   NUM  *   *
+--     CLM_NON_UTLZTN_DAYS_CNT      NUM  *   *
+--     NCH_BLOOD_PNTS_FRNSHD_QTY    NUM  *   *
+--
+alter table snf_claims_new 
+    alter column CLM_UTLZTN_DAY_CNT           ${logic.alter-column-type} numeric;
+alter table snf_claims_new 
+    alter column BENE_TOT_COINSRNC_DAYS_CNT   ${logic.alter-column-type} numeric;
+alter table snf_claims_new 
+    alter column CLM_NON_UTLZTN_DAYS_CNT      ${logic.alter-column-type} numeric;
+alter table snf_claims_new 
+    alter column NCH_BLOOD_PNTS_FRNSHD_QTY    ${logic.alter-column-type} numeric;
+
+-- carrier_claim_lines table:
+--     REV_CNTR_UNIT_CNT            NUM  *   *
+--     REV_CNTR_NDC_QTY             NUM  *   *
+--
+alter table snf_claim_lines_new 
+    alter column REV_CNTR_UNIT_CNT  ${logic.alter-column-type} numeric;
+alter table snf_claim_lines_new     
+    alter column REV_CNTR_NDC_QTY   ${logic.alter-column-type} numeric;
```

Comment:
Looks like we are reverting all but clm_id and clm_line_num, right?

Given that, I think that it is more straightforward and easier to digest if we simply copy the V63 over to V66, make the fixes, and add a drop table at the beginning.

Reply:
agreed. dropped the tables an completely re-created